parm/config/config.base.emc.dyn

```diff
@@ -299,7 +299,7 @@ export gldas_cyc=00
 # fi
 
 # Microphysics Options: 99-ZhaoCarr, 8-Thompson; 6-WSM6, 10-MG, 11-GFDL
-export imp_physics=11
+export imp_physics=8
```

Comment:
Then please point me to the tests I'm supposed to do.  The requirements for what I'm supposed to test is not documented anywhere.  In the past, @WalterKolczynski-NOAA has done any cycled tests that has been required.

Reply:
Will likely discuss this in tomorrow's g-w CM tag-up and come up with a test plan. Stay tuned.

> The requirements for what I'm supposed to test is not documented anywhere.

You're right, we need to add requirements to the wiki and/or PR template. This was discussed today by the CMs and we'll work to flesh out those requirements for future PRs.

Reply:
They need to be tested, but I wanted an evaluation of the settings for general use before we even move on to that point.

I don't mind doing setting up the cycling tests, but here is how I've been setting up my cycling tests if you want to do them:

1. Go through the normal checkout/build/link. Make sure you do build_all without the -c flag (you can still use coupled with the link script).
2. Modify this as needed for your own paths for experiment setup (4½ cycle test):
```
./setup_expt.py cycled --app ATM --resdet 384 --resens 192 --comrot $NOSCRUB/global-workflow --expdir $SAVE/global-workflow/expdir --idate 2020083000 --edate 2020083112 --nens 80 --gfs_cyc 4 --start warm --pslot $PSLOT
```
3. Go to comrot, delete the $PSLOT directory that was created and then use this command to copy the warm-start files:
`cp -as $SOURCE_DIR $PSLOT`

Where SOURCE DIR is (the trailing slash is important):
hera: `/scratch2/NCEPDEV/ensemble/noscrub/Walter.Kolczynski/global-workflow/C384_warm_IC/`
orion: `/work2/noaa/global/wkolczyn/noscrub/global-workflow/C384_warm_IC/`
dell: `/gpfs/dell2/emc/modeling/noscrub/Walter.Kolczynski/global-workflow/C384_warm_IC/`

This should copy the directory _structure_ of the source, but all the files will be symlinks.

4. [Optional to speed things up] In the experiment directory:
  a. In config.base, edit FHMAX_GFS to be 24.
  b. In config.resources, edit wtime_fcst_gfs to be "01:00:00"
5. Run `setup_workflow.py` as normal
6. [Optional] Edit the workflow XML
  a. Change the MAXTRIES to 1, so if there is a failure it doesn't try to do it twice
  b. Change TASKTHROTTLE to "80" so more EnKF jobs run at once
  
Normally I'm just looking for successful completion, but with changes like these (particularly to Thompson MP), we should get someone from the GDAS team to evaluate.

Reply:
This [document](https://docs.google.com/document/d/10GJq14G03oqAmQgMJXoCWCPcjM8j9fDlqf5zRoayr9o/edit?usp=sharing) has been linked in the Wednesday Coupled Model Meeting notes since Feb 16th. We've been talking about this PR since about that time.  So I would hope that these settings are in fact settled and nothing is coming as a surprise to anyone.  I make one change of settings today based off of a misunderstanding I had and the fact that I had only been able to test with aerosols since Friday.   Also, please know that all changes to the settings were made with @yangfanglin awareness and help.  

Thank you for providing instructions for the cycled test @WalterKolczynski-NOAA.  That's great information to have for the future.  Are we far enough along in the settings changes to get start coordinating the evaluation of the impact to DA?

Reply:
Sure, those settings are fine _for the prototype_ (though I still wonder what is going on with gwd_opt). But did any of the physics group vet them for knock-on effects of applying it to everything? AFAIK the discussions have primarily been in the context of the next prototype, not the GFS system at-large. I'm just worried about tunnel vision on the prototypes and wrecking someone else using global-workflow.

Reply:
@JessicaMeixner-NOAA @WalterKolczynski-NOAA  could you please point me to an input.nml file for me to check ? I did verify once with Jessica.   As for the GWD options, the physics group did make a decision to use (revert back to) the GFS.v16 setting for p8b.  As a heads up, more GWD changes are coming for p8c.    As fore workflow, if the develop (head) branch needs to support all systems including current operation, then the coupled model development should be done in a branch.

Reply:
An input.nml file can be found here: /scratch2/NCEPDEV/climate/Jessica.Meixner/p8b/prtests/coupled/aero2d/COMROOT/aero2d/gfs.20130401/00/atmos/input.nml 

After the monumental lift of getting the fully coupled atm-aero-ocn-ice-wav model into the develop branch, I'd really hope that the coupled model development will not be pushed off onto another branch.   Apologies if I misunderstood @yangfanglin but I actually went to the extra effort to ensure that the standalone and coupled model configurations were only different where they should be such as for "cpl*" settings and the value of minimum ice fraction etc.   I thought the plan was that we were all moving forward on the develop branch together and I really hope that is still the case.

Reply:
The global-workflow team is definitely against going back to maintaining another branch for prototype work.

CC: @aerorahul

Reply:
@JessicaMeixner-NOAA @WalterKolczynski-NOAA  I agree with you the development should be happening in the develop branch.  Walter referred to the setbacks to other systems.   I'd say it will get increasing difficult to maintain backward combability with legacy systems like GFS.v16 which is even still running with the IPD physics.   I am sure the workflow team will find the best way to balance the support for existing system with the need of moving the development forward.  

The back-and-forth change of physics options is part of the development process, the "Physics Wheel of Pain" as Brad Ferrier depicted when he was working at EMC.   We will work with the workflow team to eventually set up everything thing correctly.  This is part of the physics integration.

Reply:
> /scratch2/NCEPDEV/climate/Jessica.Meixner/p8b/prtests/coupled/aero2d/COMROOT/aero2d/gfs.20130401/00/atmos/input.nml

@JessicaMeixner-NOAA   The settings for the atmos and land components in your input.nml look to be correct.

Reply:
> I'd say it will get increasing difficult to maintain backward combability with legacy systems like GFS.v16 which is even still running with the IPD physics. I am sure the workflow team will find the best way to balance the support for existing system with the need of moving the development forward.

As far as I am aware, we are not continuing to support a GFSv16-like system in develop, that's purely in the `operations` branch and it's sibling branches (`feature/ops-wcoss2` and `dev_v16`, the latter of which will maintain support for GFSv16 on our tier-1 platforms). The `develop` branch is moving forward for GFSv17. As @WalterKolczynski-NOAA and @aerorahul have said, we need to know that incoming changes like this PR have been tested in all major modes and do not break things like cycling. Since no cycling tests were done yet the concern was raised.

Reply:
It is certainly not my intention to break cycling.    I will run cycled tests and coordinate with someone in the DA group to ensure they sign off on the changes.    

Please let me know how I can better understand what will be required of PRs to the workflow in the future.  The changes here have been advertised and discussed in the coupled modeling tag ups every Wednesday for a month + in which @KateFriedman-NOAA and @WalterKolczynski-NOAA are active participants at the meeting.  There will be more changes coming like this and knowing what will be required will assist in planning efforts.

Reply:
Note that DA development might be required to update both workflow and code (GSI) to run gdas forecast and data assimilation with updated physics options such as Thompson MP, Noah-MP, and RRTMGp radiation (to be included in p8c) etc.  It might be getting more and more difficult to maintain the cycling capability but still use the GSI and its associated scripts developed for GFS.v16.